### PR TITLE
WIP - OADP 3354 Remove Tech-Preview language from OADP 1.3 Data Mover

### PR DESCRIPTION
**Pull request by CarmiWisemon**
OADP 1.3.1

OCP 4.13+

Resolves - https://issues.redhat.com/browse/OADP-3354

Deploy preview - https://71871--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/about-oadp-1-3-data-mover

### CODE REVIEW

File: snippets/datamover-1-3-technology-preview.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// When including this file, ensure that {FeatureName} is set immediately before
+// the include. Otherwise it will result in an incorrect replacement.
+
+[IMPORTANT]
+====
+[subs="attributes+"]
+OADP 1.3 Data Mover moves from being a Technology Preview feature only, to being GA from OADP 1.3.1. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
```
Comment (Srivaralakshmi) [inline snippets/datamover-1-3-technology-preview.adoc:7]: @CarmiWisemon Suggest removing tech preview language out of here too.

```suggestion
OADP 1.3 Data Mover moves from being a Technology Preview feature only, to being GA from OADP 1.3.1.
```

File: snippets/datamover-1-3-technology-preview.adoc [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+====
```
Comment (Srivaralakshmi) [inline snippets/datamover-1-3-technology-preview.adoc:10]: @CarmiWisemon Suggest removing tech preview language out of here too.

File: backup_and_restore/application_backup_and_restore/installing/about-oadp-1-3-data-mover.adoc [hunk 3]
```diff
@@ -13,8 +13,7 @@ OADP supports CSI snapshots on the following:
 * Red Hat OpenShift Data Foundation
 * Any other cloud storage provider with the Container Storage Interface (CSI) driver that supports the Kubernetes Volume Snapshot API
 
-:FeatureName: The OADP built-in Data Mover
-include::snippets/technology-preview.adoc[]
+include::snippets/datamover-1-3-technology-preview.adoc[]
```
Comment (Srivaralakshmi) [inline backup_and_restore/application_backup_and_restore/installing/about-oadp-1-3-data-mover.adoc:16]: @CarmiWisemon Suggest removing the tech preview snippet and retain the following statement as an IMPORTANT admonition in the assembly instead:

OADP 1.3 Data Mover moves from being a Technology Preview feature only, to being GA from OADP 1.3.1.

File: snippets/datamover-1-3-technology-preview.adoc (hunk 1 above)
Comment (Srivaralakshmi) [reply snippets/datamover-1-3-technology-preview.adoc:7]: Move L7 info to the assembly instead.

**Review by Srivaralakshmi — COMMENTED** [review]
@CarmiWisemon Left a few comments for your consideration. PTAL and let me know your thoughts. Thank you!

**Comment by CarmiWisemon** [comment]
@Srivaralakshmi 
Thank you for your wonderful suggestion. I implemented it.
Thank you!!!

File: snippets/datamover-1-3-technology-preview.adoc [hunk 4]
```diff
@@ -0,0 +1,10 @@
+// When including this file, ensure that {FeatureName} is set immediately before
+// the include. Otherwise it will result in an incorrect replacement.
+
+[IMPORTANT]
+====
+[subs="attributes+"]
+OADP 1.3 Data Mover moves from being a Technology Preview feature only, to being GA from OADP 1.3.1.
+====
+// Undefine {FeatureName} attribute, so that any mistakes are easily spotted
+
```
Comment (Srivaralakshmi) [inline snippets/datamover-1-3-technology-preview.adoc:10]: @CarmiWisemon Do we need this snippet anymore? Or can we do without it?

**Comment by bergerhoffer** [comment]
The `branch/enterprise-4.18` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.17. And any PR going into `main` must also target the latest version branch (enterprise-4.18).

If the update in your PR does NOT apply to version 4.18 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Pull request closed without merging**